src/tk/widget/tabwin.c

```diff
@@ -115,9 +115,16 @@ unsigned int *window_addTabbedWindow( unsigned int wid, const int x,
       /* Create windows with flags.
        * Parent window handles events for the children.
        */
-      wgt->dat.tab.windows[i] = window_createFlags(
-         tabnames[i], tabnames[i], wx, wy, ww, wh,
-         WINDOW_NOFOCUS | WINDOW_NORENDER | WINDOW_NOINPUT | WINDOW_NOBORDER );
+      int flags = WINDOW_NOFOCUS | WINDOW_NORENDER | WINDOW_NOINPUT;
+      /* For windows with upper tabs, keep borders to let them overwrite hiden
+       * tabs */
+      if ( strcmp( name, "tabLand" ) && strcmp( name, "tabInfo" ) &&
```

Comment:
I don't think hardcoding names is a good idea here. Would be better to pass some parameter instead or it'll likely bite us in the ass in the future.